src/util.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit: IMHO, I don't usually love "util" modules as they usually become a dumping ground for everything.  Given that this one isn't a dumping ground I'm less apt to push to change it.  But, given that this module revolves around block device size, perhaps we name it as such?